code/src/App.js

```diff
@@ -1,9 +1,27 @@
 import React from 'react'
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
```

Comment:
if you're using semicolons here, you should be consistent and use them everywhere...